src/features/CourtCaseFilters/CourtCaseFilterWrapper.tsx

```diff
@@ -18,44 +16,38 @@ const CourtCaseFilterWrapper: React.FC<Props> = ({
   paginationBottom
 }: Props) => {
   const [areAppliedFiltersShown, setAreAppliedFiltersShown] = useState(false)
-  const classes = useCustomStyles()
   return (
     <>
-      <div className={`${classes["top-padding"]} moj-filter-layout`}>
```

Comment:
looking at the MoJ filter pattern examples, they all have a containing div with the `moj-filter-layout` class, whereas this PR has removed the parent`moj-filter-layout` div. Do we need it?

Reply:
My bad, I removed the whole div instead of removing just the class "top-padding"